tests/queries/0_stateless/03202_system_load_primary_key.sql

```diff
@@ -0,0 +1,11 @@
+-- Test to ensure SYSTEM LOAD PRIMARY KEY works as expected
```

Comment:
Please update the docs on https://clickhouse.com/docs/en/sql-reference/statements/system (the corresponding markdown file is part of this repository)